src/appdistribution/client.ts

```diff
@@ -24,6 +25,16 @@ export enum UploadStatus {
   ERROR = "ERROR",
 }
 
+/** Enumn representing the App Bundles state for the App */
+export enum AabState {
+  AAB_STATE_UNSPECIFIED = 1,
+  ACTIVE,
+  PLAY_ACCOUNT_NOT_LINKED,
+  NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT,
+  APP_NOT_PUBLISHED,
+  AAB_STATE_UNAVAILABLE,
```

Comment:
We should also add `PLAY_IAS_TERMS_NOT_ACCEPTED`

Reply:
👍

Reply:
Done!